style.css

```
@@ -0,0 +1,1849 @@
+/* =========================================================
+   COMPUTER
+========================================================= */
+
+/*:root {
+    --comp-body: #a5a3ab;
+    --comp-body-dark: #8c8a92; /* For 3D sides */
+    --comp-border: #121214;
+    --comp-screen: #062f3a;
+    --comp-screen-border: #5e5f67;
+    --comp-accent: #11e5e0;
+}
```

Stylelint (17.14.0)

[error] Unexpected } (CssSyntaxError)

(CssSyntaxError)